src/workspaces/cookiefactoryv2/README.md

```diff
@@ -56,21 +104,59 @@ This project walks you through the process of setting up the Bakersville Cookie
       ```
     - Open scene in console: `https://us-east-1.console.aws.amazon.com/iottwinmaker/home?region=us-east-1#/workspaces/__FILL_IN__/scenes/CookieFactory`
 
-4. Setup Web Application
-   - Follow instructions in [CookieFactoryDemo](./CookieFactoryDemo/README.md)
+
+
+## Configure Web Application
+
+First change directory to CookieFactoryDemo
+``` 
+cd ../CookieFactoryDemo 
+```
+
+1. Change `WORKSPACE_ID` in `src/config/sites.ts` to your workspace ID.
+2. Use the following CLI command to administratively set the password for your demo Cognito user, Fran. Your User_Pool_id can be found after your CDK deploy completed in the CloudFormationOutput (printed to console). The demo Username is "fran@cookiefactory". Be sure to create a password that meets the default Cognito password requirements (Lowercase letter, Uppercase letter, Number, Symbol, Length >= 8)
+```
+aws cognito-idp admin-set-user-password --user-pool-id "[YOUR_USER_POOL_ID]" --username "[USERNAME]" --password "[PASSWORD]" --permanent
+```
+
+3. Set your AWS and Cognito user credentials in: `src/config/cognito.template.ts` and `src/config/user.template.ts` and rename the files to `src/config/cognito.ts` and `src/config/users.ts`, respectively. You will need your newly created Cognito UserPoolId, IdentityPoolId, ClientId, Username and password for your user
+
+OPTIONAL: view [Console instructions](./CookieFactoryDemo/COGNITO_SAMPLE_SETUP_CONSOLE.md) for sample Amazon Cognito configuration (already created through CDK). You may use Cognito to configure custom security settings for your User Pools.
+
+### Install
+
+```shell
+npm install
+```
+
+### Development server
+
+* Edit `webpack.dev.js` if needed to change the port for the application (default is port 5000)
+
+```shell
+npm run dev
+```
+
+* Navigate to the application at `localhost:5000` (default) - Note: it may take a minute to load the first time
+
+### Build
+
+```shell
+npm run build
+```
+---
 
 ## Cleanup
```

Comment:
fyi: I forgot to include workspace deletion steps in the original README, was thinking to provide console steps and tmdt steps (the `destroy` command worked well for me btw!) -- feel free to include if you have bandwidth (no worries if not)

Reply:
"console and tmdt", as in both options?

Reply:
right, I just meant as 2 different ways to delete the workspace-related resources

creation in console is fairly straightforward IMO but deletion has some extra steps so tmdt may be easier, but since we have console to create we should probably be consistent and at least include console to delete too

Reply:
Makes sense. Should we include tmdt workspace create too? This would require either passing in a blank TMDT file as --dir or add a partial version of that code directly in this folder.

Reply:
sounds good, let's give it a try and have the console steps as an alternative, we can revise based on customer feedback